lms/templates/verify_student/make_payment_step.underscore

```diff
@@ -77,6 +78,13 @@
           <% } %>
         <% } %>
       </ul>
+
+      <p><%= gettext("Once payment has been processed, you commit to taking a remotely proctored exam following the protocol defined for this course. Please check that your equipment is compatible with the proctoring system of our ProctorU provider by clicking <a href='http://www.proctoru.com/testitout/' target='_blank'>this link</a>.") %></p>
+      <p>
+        <input class="understand-requirement" type="checkbox" style="display: inline;" /> <%- gettext("I wish to take an exam to validate this course and obtain a certificate") %><br />
+        <input class="understand-requirement" type="checkbox" style="display: inline;" /> <%- gettext("I can schedule the remote exam at the dates defined for this course") %><br />
+        <input class="understand-requirement" type="checkbox" style="display: inline;" /> <%- gettext("My computer equipment is compatible with the exam proctoring tools") %><br />
+      </p>
     </div>
     <% } %>
 
```

Comment:
Lorsque je rajoute des `<label for="checkbox">` les labels ne sont pas cliquables. Je parviens à ce résultat en faisant :

    <input id="id1" ...> <label for="id1" style="display: inline;">...</label>

Le résultat est ci-dessous.
![checkboxes-labels](https://cloud.githubusercontent.com/assets/44319/13577096/206f6d96-e491-11e5-91af-a33480b8a427.png)